### ISSUE TITLE
Too low difference between enabled/disabled back/forward button

### ISSUE THREAD

**Issue by Amad22** [Aris-t2/CustomCSSforFx#430]
Hello
Thank you for useful and updated Fx addon.
I often need to look if I the actual tab is go-back-able or not (so the last link was opened in (this) new tab)
and so if schall I press backspace or ctrl+w (close tab) for return to previous page.
It suffices to look at the back/forward button on toolbar, if the back button is enabled or disabled.
But the contrast between enabled and disabled state is very small for first look. 
I have worked around it by using Fx-3 back/forward button but it could be treated direct in original icons.
The appearance of the current toolbar icons is generally too "thin" and indistinct. Maybe it's because of the ability to use both light and dark modes. But it always takes me a "while" to find the right button I want, compared to the classic full icons. Maybe I will use Fx-3 all toolbar icons. 
I'm just reporting feedback, maybe you take into account in some future version or not.
![prntscrn](https://user-images.githubusercontent.com/68122450/146069653-0a2bff1a-1ad8-4730-b58a-b3f3c4833933.png)

**Comment by Aris-t2** [comment]
You are right, the visual difference between "enabled" and "disabled" state of those buttons is tiny in some cases. The reason for this is the opacity and grayscale values used inside the CSS file(s).

Test this, when using the green back and forward buttons seen on your screenshot:

```css
#main-window #navigator-toolbox #nav-bar :is(#back-button, #forward-button)[disabled] .toolbarbutton-icon {
  filter: grayscale(75%) !important;
}
```

**Comment by Aris-t2** [comment]
I will also add the 75% value to next release.

**Comment by Amad22** [comment]
Thank you. When I wrote the issue, I actually meant default b/f button in actual addon release. The printscreen with Fx3 b/f button was trying for work-around. So if you will do some change about this issue, tune it preferably for actual f/b button. 

I am sending printscreen for change 75%. I recommend rather 80% or 85% or another way (bleaching like by actual f/b button or combination both ways). But it is for detail only and for completeness.
![fx3_bfb_75perc](https://user-images.githubusercontent.com/68122450/147080576-e0d770f0-5998-4a9f-ba17-1e08658dbdf6.png)

I think the parallel problem is that the (actual) arrows are too thin, so basic bleach change on that thin symbol does not enought significant change. 
Buttons are little strange at all. They look modern, but they are not too practical for often use when users want to fast find the right icon; important settings-button is dull, "useless" history-button (people use ctrl+h) is most distinctive, and buttons lost their established colors (yellow star for favorits and green arrow down for download). I recommend use traditional fullfill icons instead empty. I am sorry for critics and hairsplitting, I do not mean it badly; I worked as app-user-interface-tunner previously, maybe this is side effect...
